src/components/RadioButtonWithLabel.js

```diff
@@ -55,13 +58,18 @@ const RadioButtonWithLabel = (props) => {
                     label={props.label}
                     hasError={props.hasError}
                 />
-                <TouchableOpacity
+                <PressableWithFeedback
+                    focusable={false}
+                    accessible={false}
                     onPress={() => props.onPress()}
-                    style={[styles.ml3, styles.pr2, styles.w100, styles.flexRow, styles.flexWrap, styles.flexShrink1, styles.alignItemsCenter]}
+                    style={[styles.flexRow, styles.flexWrap, styles.flexShrink1, styles.alignItemsCenter]}
+                    wrapperStyle={[styles.ml3, styles.pr2, styles.w100]}
+                    // disable hover style when disabled
+                    hoverDimmingValue={1}
```

Comment:
```suggestion
                    hoverDimmingValue={1}
                    pressDimmingValue={0.2}
```
This seems to be what the current press dimming is set to. I don't hugely care either way, so if you think it's worth changing, do it, if not, don't.

Reply:
Thats simple change so I think we can add it